test/factories/user.rb

```diff
@@ -3,7 +3,7 @@
 FactoryBot.define do
   factory :user do
     email { Faker::Internet.email }
-    password { Faker::Internet.password(min_length: 8, max_length: 128, mix_case: true) }
+    password { Faker::Internet.password(min_length: 10, max_length: 20, mix_case: true) + '!' }
```

Comment:
Do we set an upper limit on password? Why not test a maximum?
Also instead of adding an `!` we can use the option `special_characters: true`

Reply:
Agreed, changed.

Reply:
Hmm, I don't think Faker is behaving as we thing it should (and how I would assume it would based on the docs):
```
irb(main):014:0> Faker::Internet.password(min_length: 10, max_length: 126, mix_case: true, special_characters: true)
=> "*^#*$%%^^*"
```